src/components/organisms/ContentBox.jsx

```diff
@@ -157,8 +157,8 @@ const FullscreenContainer = styled.div`
   position: absolute;
   top: 0;
   left: 0;
-  height: 100vh;
-  width: 100vw;
+  height: 100%;
```

Comment:
Especially here. As far as I remember correctly the height in % is calculated based on the parents width.